c/kore/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:18.04
+
+RUN apt-get update -qq && \
+    apt-get install -qq -y \
+            wget \
+            make \
+            gcc \
+            nginx \
+            openssl \
+            libssl-dev && \
+    apt-get clean -qq -y && \
+    apt-get autoclean -qq -y && \
+    apt-get autoremove -qq -y
+
+RUN wget -c https://kore.io/releases/kore-3.1.0.tar.gz && tar xvf kore-3.1.0.tar.gz
+RUN cd kore-3.1.0 && TASKS=1 NOTLS=1 make && make install
+
+WORKDIR /usr/src/app
+
+COPY hello hello
+
+WORKDIR /usr/src/app/hello
+
+RUN pwd
+
+RUN kodev build
+
+RUN rm -fr /etc/nginx/sites-enabled/default
+
+RUN echo 'upstream backend {\n\
```

Comment:
Is including nginx in this really necessary since there is only one C application running anyway (with hopefully multiple threads).  Seems superfluous and makes this test overly bigger than it really needs to be.

Reply:
Hi! As I described in the git commit message, the dynamic `/user/:id` path pattern is not supported yet. I have read all the documentations and examples of Kore but did not find any related information.

* Documentations: https://docs.kore.io/3.0.0/
* Change log: https://kore.io/releases/3.1.0#changelog
* Examples: https://github.com/jorisvink/kore/tree/master/examples

So there must be something provided to do the magic if that path pattern should be implemented. And I guess nginx would be the best choice at the moment.
Or we should wait for the implementation by Kore itself.

Reply:
Or there are something I am missing...

Reply:
>  the dynamic /user/:id path pattern is not supported yet.

You don't need it, the style of many of these frameworks is to do a prefix match then perform further parsing on the remaining path to get the `:id` part.  :-)